server/public/styles.css

```diff
@@ -0,0 +1,47 @@
+html,
+body {
+  margin: 0 auto;
+  padding: 0;
+  height: 100%;
+  max-width: 575px;
+}
+
+.justify-content-center {
+  justify-content: center;
+}
+
+.header-container {
+  font-family: 'Actor', sans-serif;
+  color: #ff4b3a;
+  text-align: center;
+}
+
+.header {
+  font-size: 2.5em;
+  margin: 1rem;
+  margin-top: 1rem;
+  margin-bottom: 0.5rem;
+}
+
+.sub-header {
+  font-size: 1.5rem;
+  margin-top: 0;
+}
+
+.restaurant-container {
+  width: 575px;
+}
```

Comment:
Please look at Bootstrap's helper's and remove as much of this CSS as you can.

https://getbootstrap.com/docs/4.5/utilities/flex/
https://getbootstrap.com/docs/4.5/utilities/text/
https://getbootstrap.com/docs/4.5/utilities/spacing/